CONTRIBUTING.md

```diff
@@ -18,8 +18,13 @@ Follow either of the two links above to access the appropriate CLA and instructi
 1. Submit an issue describing your proposed change to the repo in question.
 2. The [repo owners](OWNERS) will respond to your issue promptly.
 3. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
-4. Fork the desired repo, develop and test your code changes.
-5. Submit a pull request.
+4. Before starting to work on an issue, please mark the issue with the `lifecycle/active` label or contact a Kubeadm maintainer to do so;
```

Comment:
given this is an entry level guide, i'd add notes on how to actually apply/remove the label:

> To add the label type `/lifecycle active` and to remove it type `/remove-lifecycle active` in a comment on an issue.